### ISSUE TITLE
[EN]Valor Cap being buggy

### ISSUE THREAD

**Issue by Luna-maria** [Atlantiss/BugTracker#5005]
Making a ticket because one of the gms suggested doing so. Issue might already be known but he said to post it so it won't be forgotten. 

Two weeks in a row now I've been 20 vps short of the vp cap, not because I got vps solely through heroics, but because for some odd reason, killing raid bosses (Alakir one week, and Chimaeron the other week) wouldn't give me the remaining 20 vps I was short of the cap

I've relogged, cleared cache, tried a random heroic, nothing would get me the last 20 points to fill up the valor cap. 

It's not much but if it happens every week well, we can all do the maths that it'll eventually prevent me from getting stuff a week faster

**Comment by LordKnightCecilo** [comment]
After experimenting a bit. It seems that the first 980 VP you earn will disable your amount of RHCs that award VP. So.. if you do say 140 VP from raids. And then do 6 more heroics. You will not get the final heroic reward, it believes you have already done 7 for the week. In short, it doesn't seem to count the amount of RHCs you have completed, It will base it off of how much VP You have. Or at least this is how it seems from my experimenting with my characters. 

Ideally this means you would get all 7 heroics done, and then do a raid.

**Comment by Thefakemccoy** [comment]
@LordKnightCecilo you are right. Its counting VP total not amount of heroics completed. It shouldnt but it does. So yes do all 7 heroics first then do your BH or a raid and you will 1k cap.

If
